tests/core/test_daemon_rpc.py

```diff
@@ -5,11 +5,17 @@
 from chia.daemon.client import connect_to_daemon
 from tests.setup_nodes import bt
 from chia import __version__
+from tests.util.socket import find_available_listen_port
+from chia.util.config import save_config
 
 
 class TestDaemonRpc:
     @pytest_asyncio.fixture(scope="function")
     async def get_daemon(self):
+        bt._config["daemon_port"] = find_available_listen_port()
+        # unfortunately, the daemon's WebSocketServer loads the configs from
+        # disk, so the only way to configure its port is to write it to disk
+        save_config(bt.root_path, "config.yaml", bt._config)
```

Comment:
it's still a problem that `bt` is a global variable, shared between all tests. But that will hopefully be fixed in a separate PR.

Reply:
https://github.com/Chia-Network/chia-blockchain/pull/10358